mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -113,35 +113,30 @@ view: +subscriptions {
   }
 
   dimension: billing_grace_period {
-    type: number
```

Comment:
Wouldn't the type still be relevant here?

Reply:
the dimension is actually an `INTERVAL`, which looker doesn't directly support